jboss-kie-smartrouter/added/launch/jboss-kie-smartrouter.sh

```diff
@@ -47,6 +48,22 @@ function configure_mem_ratio() {
     export JAVA_INITIAL_MEM_RATIO=${JAVA_INITIAL_MEM_RATIO:-25}
 }
 
+function configure_logger_config_file() {
+  # JUL implementation: https://docs.oracle.com/javase/7/docs/api/java/util/logging/Level.html
+  local allowed_log_levels=("ALL" "CONFIG" "FINE" "FINER" "FINEST" "INFO" "OFF" "SEVERE" "WARNING")
```

Comment:
the indentation still with 2 spaces instead 1 tab.

Reply:
fixed